glossary/general/library.md

```diff
@@ -0,0 +1,3 @@
+# Library
+
+Libraries are built-in collections of functions that we can use within our code to make the job easier and save time.
```

Comment:
```suggestion
Libraries are collections of functions that we can use within our code to make the job easier and save time. There are built-in libraries that are always available to use after import, and there are some created by the community.
```

I think saying all are built-in might not be good as languages like Python have many community-driven ones that need to be installed before use